packages/go_router/lib/src/route.dart

```diff
@@ -1032,6 +1048,21 @@ class StatefulShellBranch {
   /// The observers parameter is used by the [Navigator] built for this branch.
   final List<NavigatorObserver>? observers;
 
+  /// Whether this route branch should be loaded only when navigating to it for
```

Comment:
Can you inverse the statement? from reading it, it seem to not preload when this property is true

Reply:
Right, forgot to change that. How about this: "Whether this route branch should be eagerly loaded when navigating to it for the first time..."?

Reply:
No, that's also confusing, it's not preloading when accessing the branch for the first time but when accessing the shell.

"Whether this "StatefulShellBranch" should be preloaded once the "StatefulShellRoute" has been accessed."

Reply:
Ah, but of course, how about this: 
"Whether this route branch should be eagerly loaded when navigating to the associated StatefulShellRoute for the first time."

Reply:
Yes that's much better

Reply:
yes sounds good to me, can you reflect that to the doc string?